exercise1solution/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace exercise1solution
+{
+    class Program
+    {
+        static List<int> PrimeNumbers(int n)
+        {
+            bool IsPrme(int number)
+            {
+                if (number < 2)
+                    return false;
+                for (int i = (int)Math.Sqrt(number); i >= 2; i--)
+                {
+                    if (number % i == 0)
+                    {
+                        return false;
+                    }
+                }
+                return true;
+            }
+
+            var primeNumbers = new List<int>();
+            for (var i = 2; i < n; i++)
+            {
+                if (IsPrme(i))
+                {
+                    primeNumbers.Add(i);
+                }
+            }
+            return primeNumbers;
+        }
+        static void Main(string[] args)
+        {
+            Func<int, List<int>> createList = PrimeNumbers;
```

Comment:
No need to have `createList`, just need to call directly `PrimeNumbers(number).ForEach(Console.WriteLine);`

Reply:
done